python-service/services/scanner.py

```
@@ -0,0 +1,85 @@
+        _add(t, seen, candidates)
+
+    # 2. Upcoming earnings in the next 5 days (high-signal catalyst)
+    today = date.today()
+    from_date = today.isoformat()
+    to_date = (today + timedelta(days=5)).isoformat()
+    try:
+        earnings = await finnhub.get_earnings_calendar(from_date, to_date)
+        for item in earnings[:20]:
+            _add(item["symbol"], seen, candidates)
+    except Exception:
+        pass
```

Ruff (0.15.4)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)